src/model/operator.ts

```diff
@@ -25,14 +25,16 @@ export const getRelevantOperators = async (onlyRelevantOperators: boolean) => {
     'קווים',
     'תנופה',
   ]
-  const agencyMap = new Map()
+  const agencyMap: Map<string, Agency> = new Map()
   agencyList.forEach((obj) => {
-    agencyMap.set(obj.agency_name, obj)
+    if (obj) {
+      agencyMap.set(obj.agency_name, obj)
+    }
   })
 
   const res = relevant.reduce((acc: Operator[], name: string): Operator[] => {
     return agencyMap.has(name)
-      ? [...acc, { name, id: agencyMap.get(name)?.operator_ref.toString() }]
+      ? [...acc, { name, id: (agencyMap.get(name) as Agency).operator_ref.toString() }]
```

Comment:
is `as Agency` necessary?

Reply:
If we leave it as is it won't be assignable:
```
Type '{ name: string; id: string | undefined; }[]' is not assignable to type 'Operator[]'.
  Type '{ name: string; id: string | undefined; }' is not assignable to type 'Operator'.
    Types of property 'id' are incompatible.
      Type 'string | undefined' is not assignable to type 'string'.
        Type 'undefined' is not assignable to type 'string'.
```

Reply:
```suggestion
      ? [...acc, { name, id: (agencyMap.get(name)!).operator_ref.toString() }]
```
will tell it that it can't be undefined